### PR TITLE
NC-152 disabled week and month periods.

### PR DESCRIPTION
**Pull request by drakonen**
@kevinvandijk @aaronz8

### CODE REVIEW

**Comment by Juice10** [comment]
@drakonen Why are you taking out the defaults?

**Comment by drakonen** [comment]
@Juice10 https://bottlenose.myjetbrains.com/youtrack/issue/NC-223

File: src/models/period.js [hunk 1]
```diff
@@ -205,13 +205,16 @@ define(['core', './helpers/timezone-detector'], function(Openhose, timezoneDetec
...
       var end = Openhose.moment(moments.end + 1); // correct the 1ms offset on the end
 
-      if (end.diff(start, 'years') >= 2) {
-        return 'month';
-
-      } else if (end.diff(start, 'months') >= 3) { // FIXME: this fails on summertime switch
-        return '1w';
-
-      } else if (end.diff(start, 'days') >= 14) {
+      // Do not use month and week for now.
```
Comment (kevinvandijk) [inline src/models/period.js]: Can you comment the youtrack url here?

**Comment by kevinvandijk** [comment]
:+1: :beers:

**Pull request merged**